boards/librem_l1um_v2/librem_l1um_v2.config

```diff
@@ -42,7 +41,6 @@ export CONFIG_BOOT_KERNEL_REMOVE="plymouth.ignore-serial-consoles"
 export CONFIG_BOOT_RECOVERY_SERIAL="/dev/ttyS0"
 export CONFIG_BOOT_DEV="/dev/nvme0n1p1"
 export CONFIG_BOARD_NAME="Librem Server L1UM v2"
-export CONFIG_AUTO_BOOT_TIMEOUT=5
```

Comment:
Why was this deleted?

Reply:
> Why was this deleted?
Manual removal of lines returned from a grep. my bad. Corrected under [2655c4c](https://github.com/linuxboot/heads/pull/1515/commits/2655c4cc4d4dcfe10910ae6ad666f3b4798cf700)

Reply:
I reverted changes for unifying tpmr counter_create to leave things to be refactored further later on in seperate PR and made the fix under last commit [9e0491e](https://github.com/linuxboot/heads/pull/1515/commits/9e0491e9dbd68ffdd6d15aa398addd16738fcfd9). Sorry about that.